libs/forecasting/src/lib/ForecastingDashboard/ForecastingDashboard.styles.ts

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import {
+  IStyle,
+  mergeStyleSets,
+  IProcessedStyleSet,
+  getTheme,
+  FontWeights
+} from "@fluentui/react";
+import { descriptionMaxWidth, flexLgDown } from "@responsible-ai/core-ui";
+
+export interface IForecastingDashboardStyles {
+  dropdown: IStyle;
+  sectionStack: IStyle;
+  configurationActionButton: IStyle;
+  topLevelDescriptionText: IStyle;
+  descriptionText: IStyle;
+  generalText: IStyle;
+  generalSemiBoldText: IStyle;
+  selections: IStyle;
+  smallDropdown: IStyle;
+  mediumText: IStyle;
+  forecastCategoryText: IStyle;
+  subMediumText: IStyle;
+  smallTextField: IStyle;
+  errorText: IStyle;
+}
+
+export const forecastingDashboardStyles: () => IProcessedStyleSet<IForecastingDashboardStyles> =
+  () => {
+    const theme = getTheme();
+    return mergeStyleSets<IForecastingDashboardStyles>({
+      configurationActionButton: {
+        marginTop: "25px"
+      },
+      descriptionText: {
+        color: theme.semanticColors.bodyText,
+        maxWidth: descriptionMaxWidth
+      },
+      dropdown: {
+        width: "400px"
```

Comment:
please help check reflow cases. As fixed widths can cause issues in 320*256 screens, most of the time it creates unwanted horizontal scroll bar. thanks.

Reply:
Couldn't agree more! In my experience, dropdowns have proven tricky in this regard as they don't seem to be flexible based on their input params. If I didn't know fluent UI I would have expected something like min, max, and default width, but I have only the one input param. Do you have any suggestions on how to handle that? I don't want it to be too short by default because it results in poor readability of the potentially long time series names.

Reply:
We can add this conditionally for large screens as in WhatIfTab.styles
 selectors: {
          "@media screen and (min-width: 1024px)": {
            paddingRight: legendWidth
          }
        }

Reply:
Great! I made it 
```
dropdown: {
        width: "auto",
        selectors: {
          "@media screen and (min-width: 1024px)": {
            width: "600px"
          }
        }
      },
```
so for small screens it's just the full width and as soon as we have a larger screen it's 600px. I know that's pretty wide but the time series are named after their id column names + values, e.g.,
![image](https://user-images.githubusercontent.com/10245648/211703335-35706db1-7f45-41de-b66c-1bf0b71f9868.png)
so this can potentially get quite long. I've made it so that it's not longer than the text above which makes it look nice visually IMO.

Reply:
I'm assuming this solves the problem, marking as resolved, but feel free to reopen if it's not what you were thinking of!